src/javascript/app/pages/cashier/cashier.js

```diff
@@ -19,7 +23,7 @@ const Cashier = (() => {
         let $toggler;
         if (anchor) {
             $toggler = $(`[data-anchor='${anchor}']`);
-            $toggler.find('.td-description').addClass('active'); // toggle open description
+            $toggler.find('.td-descriptioncashier_iframe').addClass('active'); // toggle open description
```

Comment:
why do we need to rename this?

Reply:
back to previous version, thanks. No need to do it